src/main/rx/rx.c

```diff
@@ -363,6 +364,8 @@ void rxInit(void)
     // Setup source frame RSSI filtering to take averaged values every FRAME_ERR_RESAMPLE_US
     pt1FilterInit(&frameErrFilter, pt1FilterGain(GET_FRAME_ERR_LPF_FREQUENCY(rxConfig()->rssi_src_frame_lpf_period), FRAME_ERR_RESAMPLE_US/1000000.0));
 
+    pt1FilterInit(&rsnrFilter, 0.25f);  //just a bit of filtering to remove excessive jumpiness
```

Comment:
Maybe it would be simpler, and provide user-adjustability, to use the existing `rssi_src_frame_lpf_period` filter value?

The cli name could then be changed to `rssi_lpf_period` to reflect the more generic application of the cli parameter.

Reply:
We should also do the same for the link quality element, i.e, use the PT1 set by `rssi_src_frame_lpf_period` in place of `updateLinkQualitySamples`.

This way we have the same adjustable filtering for link quality, RSSI, and RSSI dBm.  And we can remove the median filtering methods completely.

Reply:
rssi_src_frame_lpf_period is only used for smoothing frame errors, which in turn is only used by SBUS implementation, which at the very least means this code shouldn't be in rx.c.
LQ is a late indicator, if it's not 100 when I'm flying, I want to notice asap, so in my case I'd prefer not to smooth it.
Another problem with smoothing rssi/snr is that OSD warnings then are not reacting to peaks, and since telemetry is sending back the unsmoothed values to your tx, you will get warnings on your radio while the OSD shows (smoothed) safe values without warnings (this issue puzzled me for a while until I figured what was going on). Having it configurable at least provides a way to make it more consistent (at the expense of jumpiness).
So:
- In case of providing adjustable smoothing I'd go for a simpler name such as rssi_smoothing, with a 1..100 range or something, and not include LQ. Could alternatively be rssi_rsnr_smoothing for clarity. What do you think?
- Do I move the frame error code to SBUS? or remove entirely?

Reply:
@KarateBrot there are two problems with your proposal: first we don't know rxRateHz and then even if we assume it to be around 50Hz, the useful cutoffs I've seen are around 2Hz or lower, so it's very cumbersome to work with such a compressed range. That's why the range and values I proposed were purely practical (ie: no filtering, a bit of filtering, a lot of filtering).